views/skus/edit.erb

```diff
@@ -0,0 +1,81 @@
+<div class="container">
```

Comment:
@hsolorza It seems like @alxngyn was having this issue too, it appears to be a caching problem. If you navigate to environment.erb, and change `max_age: 1 ` on line 22 and  ':cache_max_age, 1' on 23 instead of 300, they should update faster.